caffe2/CMakeLists.txt

```diff
@@ -562,12 +562,14 @@ if(NOT INTERN_BUILD_MOBILE OR NOT BUILD_CAFFE2_MOBILE)
     set_source_files_properties(${TORCH_SRC_DIR}/csrc/jit/tensorexpr/llvm_jit.cpp PROPERTIES COMPILE_FLAGS -Wno-noexcept-type)
     # Force -Werror on several files
     set_source_files_properties(${CMAKE_CURRENT_LIST_DIR}/../aten/src/ATen/native/mkldnn/Pooling.cpp PROPERTIES COMPILE_FLAGS "-Werror")
+    # Disable non-virtual-dtor error here because 3rd party (LLVM) violates that warning and fails the build
+    set_source_files_properties(${TORCH_SRC_DIR}/csrc/jit/tensorexpr/llvm_codegen.cpp PROPERTIES COMPILE_FLAGS "-Wno-error=non-virtual-dtor")
```

Comment:
I realize that I need to move this here instead of line 572 because the latter only applies for GCC > 9.0.0

Reply:
I think the problem with this approach is that `set_source_file_properties` overrides previous one